packages/analytics-js/src/constants/logMessages.ts

```diff
@@ -257,6 +257,9 @@ const INVALID_POLYFILL_URL_WARNING = (
 const BAD_COOKIES_WARNING = (key: string) =>
   `The cookie data for ${key} seems to be encrypted using SDK versions < v3. The data is dropped. This can potentially stem from using SDK versions < v3 on other sites or web pages that can share cookies with this webpage. We recommend using the same SDK (v3) version everywhere or avoid disabling the storage data migration.`;
 
+const PAGE_UNLOAD_ON_BEACON_DISABLED_WARNING = (): string =>
```

Comment:
If it is a static string value, it should not be a function. Let's define it as a string constant.

Reply:
Addressed